### PR TITLE
Many bugfixes!

### PR DESCRIPTION
**Pull request by cadyn**
Big boi bugfix hours
Closes issues #506, #501, #487, #475, and #370 

Gonna send a good few of these upstream but I did all the work on chompstation code anyways so enjoy

Edit: Github does not feel like linking issues or I'm missing something, so oh well.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/syringes.dm [hunk 1]
```diff
@@ -63,7 +63,7 @@
 		to_chat(user, "<span class='warning'>This syringe is broken!</span>")
 		return
 
-	if(user.a_intent == I_HURT && ismob(target))
+	if(user.a_intent == I_HURT && ismob(target) && user.type!=/mob/living/silicon/robot)
```
Comment (Shadow-Quill) [inline code/modules/reagents/reagent_containers/syringes.dm:66]: Please use `isrobot(user)`

Comment (Shadow-Quill) [reply code/modules/reagents/reagent_containers/syringes.dm:66]: And in the future, should you need to check a type (and a helper doesn't exist for it), use `istype(thing, /path/to/thing/to/check)

Comment (cadyn) [reply code/modules/reagents/reagent_containers/syringes.dm]: Done

**Comment by Darlantanis** [comment]
I see you've added CHOMPedit comments to some of your changes but not others. Have any of these fixes been implemented upstream? If so, would you be able to link the upstream PR's for easier traceability?

**Comment by cadyn** [comment]
> I see you've added CHOMPedit comments to some of your changes but not others. Have any of these fixes been implemented upstream? If so, would you be able to link the upstream PR's for easier traceability?

Yessir

https://github.com/VOREStation/VOREStation/pull/8823 -> Flasher and stomach & blood problems with absorption

https://github.com/PolarisSS13/Polaris/pull/7584 -> Borg fixes (yes okay I maybe fucced it up and someone else did it better but shush)

**Pull request merged**